ansible_rulebook/builtin.py

```diff
@@ -849,3 +857,13 @@ def _get_events(variables: Dict):
     elif "events" in variables:
         return variables["events"]
     return {}
+
+
+def _collect_extra_vars(variables: Dict, user_extra_vars: Dict):
+    extra_vars = user_extra_vars.copy() if user_extra_vars else {}
+    eda_vars = {}
+    for key in ["fact", "facts", "event", "events"]:
```

Comment:
@bzwei Is this wrong place to be doing this, when we add facts/fact/event/events here 
https://github.com/ansible/ansible-rulebook/blob/68b7fd2f0552d9c22001aec3b8ac9768c484fcb5/ansible_rulebook/engine.py#L353
Should we be just setting the name we want to use instead of delaying it till the end. Because if the user had passed facts/events/event/fact we would have over written them first and then changed their names here.

Reply:
@bzwei A test would also help, we can pass variables into the ruleset execution and see what the value of the events are going to be.
Also this is going to be breaking change since the events/event have moved so any of the playbooks that some of our beta users are using will also have to change. Unless we plan on supporting both for a while